src/components/pages/Offers/Offer/OfferDetails/OfferStatusBanner/OfferStatusBanner.scss

```
@@ -0,0 +1,7 @@
+.of-offer-status-banner {
+  .bi-banner {
+    border-radius: 0;
+    margin-bottom: rem(25px);
```

```suggestion
    margin-bottom: rem(24px);
```
(multiple de 4 ou 8)